lib/src/geojson.dart

```diff
@@ -494,19 +500,19 @@ class Feature<T extends Geometry> extends GeoJSONObject {
 class FeatureCollection<T extends Geometry> extends GeoJSONObject {
   FeatureCollection({this.bbox, this.features = const []})
       : super.withType(GeoJSONObjectTypes.featureCollection);
-  List<Feature<T>> features;
+  List<Feature<T?>>? features;
```

Comment:
this can just be an empty list, rather than a list or null, right? the spec still indicates it shall be a list, but can be empty.

Reply:
https://datatracker.ietf.org/doc/html/rfc7946#section-3.3
```
The value of "features" is a JSON array.
   Each element of the array is a Feature object as defined above.  It
   is possible for this array to be empty.
```